azure-devops/run_build.yml

```diff
@@ -44,7 +38,23 @@ jobs:
         vcpkgDirectory: '$(vcpkgLocation)'
         vcpkgTriplet: ${{ parameters.targetPlatform }}-windows
     - task: run-cmake@0
-      displayName: 'Run CMake with Ninja'
+      displayName: 'Build STL Support Tools'
+      enabled: true
+      condition: eq('${{ parameters.targetPlatform }}', 'x64')
+      inputs:
+        cmakeListsTxtPath: 'tools/CMakeSettings.json'
+        useVcpkgToolchainFile: true
+        configurationRegexFilter: '.*x64-Release.*'
+        buildDirectory: $(Build.ArtifactStagingDirectory)/tools
+    - task: BatchScript@1
+      displayName: 'Enforce clang-format'
```

Comment:
It is order sensitive, it must be after the vcpkg bits because that's what sets the build architecture. You can see my previous failed attempts (e.g. here https://github.com/microsoft/STL/pull/163/commits/78f3f01e6c09a005fb403624d5a50b7a1623091d ) failed because it tried to build with a broken Clang instance.

Reply:
Ok, I guess it'll run almost as early as possible then.